ush/cam/evs_check_href_files.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/ksh
 
-set +x
+set -x
 
 vday=$VDATE
 #COMINhref=/lfs/h1/ops/prod/com/href/v3.1
```

Comment:
@BinbinZhou-NOAA Can this commented out, hardcoded path be removed? Thanks!

#COMINhref=/lfs/h1/ops/prod/com/href/v3.1